libs/output-mapping/azure-pipelines.tests.yml

```diff
@@ -9,11 +9,72 @@ jobs:
 
   - template: ../../azure-pipelines/jobs/run-tests.yml
     parameters:
-      jobName: tests_php74_tableWriter
+      jobName: tests_php74_unitTests
       dependsOn: [cs]
-      displayName: Run Test Suite
+      displayName: Unit Tests
       serviceName: ci-output-mapping
-      testCommand: /code/libs/output-mapping/vendor/bin/phpunit --testsuite main
+      testCommand: /code/libs/output-mapping/vendor/bin/phpunit --testsuite unit-tests
+      variables:
+        STORAGE_API_URL: $(STORAGE_API_URL_AWS)
+        RUN_SYNAPSE_TESTS: 1
```

Comment:
Why almost of suites has `RUN_SYNAPSE_TESTS` enabled?
We have couple of tests that runs on Synapse backend and it should not be needed for them.

Reply:
Good point, it's a result of C&P 😄 .  It would probably be best to isolate `synapse/Abs` like I've done with redshift.  Thanks for the catch Erik 🍻